src/main/java/de/tum/in/www1/artemis/service/tutorialgroups/TutorialGroupChannelManagementService.java

```diff
@@ -258,11 +256,6 @@ private String determineUniqueTutorialGroupChannelName(TutorialGroup tutorialGro
             return channelName;
         }
 
-        // try to make it unique by adding a random number to the end of the channel name
-        // if already max length remove the last 3 characters to get some space to try to make it unique
-        if (channelName.length() >= 30) {
-            channelName = channelName.substring(0, 27);
-        }
         do {
```

Comment:
Why did you remove these lines?

Reply:
`channelName` is set to `tutorgroup-` and a substring at most 18 characters long. That makes the variable itself, at most, 28 characters long. So the conditional always evaluates to false.